.github/workflows/ci.yml

```
@@ -29,16 +29,29 @@ jobs:
         pip-version:
           - "latest"
           - "20.0"
+        include:
+          - os: Ubuntu
```

How about
```suggestion
          - os: Ubuntu-20.04
```
(latest is 16.04 atm)